### PR TITLE
add .gitattributes file and some unit tests

### PR DESCRIPTION
**Pull request by kryptokinght**

### CODE REVIEW

**Comment by bogas04** [comment]
@kryptokinght could you please set it such that eol is always LF for .js, .jsx, .html, .css., .ts, .tsx, .json?

**Comment by kryptokinght** [comment]
OK, will do it right now!

**Comment by kryptokinght** [comment]
Incorporated all changes that were requested

File: components/Item/__tests__/Item.test.js [hunk 1]
```diff
@@ -6,7 +6,10 @@ import Item from '../Item';
 describe('<Item />', () => {
   it('renders correctly', () => {
     const wrapper = mount(<Item name="Item one" id="123" />);
-
     expect(wrapper).toMatchSnapshot();
   });
+  it('should contain name=testing', () => {
```
Comment (bogas04) [inline components/Item/__tests__/Item.test.js]: The snapshot test will take care of this.

So general policy is that 

To test if markup is coming correctly, rely on snapshots

To test behavior (onClick, etc) use simulations.

**Comment by kryptokinght** [comment]
Removed prop testing for <Item />

**Pull request merged**